packages/extension-chakra-storefront/src/constants.js

```diff
@@ -8,40 +8,6 @@
 import {defineMessage} from 'react-intl'
```

Comment:
Most const has been moved into extension config, the rest are related to translations which we will move later when we have solution on handling translations